amplify.yml

```diff
@@ -11,9 +11,6 @@ frontend:
         - yarn install
     build:
       commands:
-        - REACT_APP_APES_AUTH_SECRET_KEY=${REACT_APP_APES_AUTH_SECRET_KEY}
-        - REACT_APP_ANGEL_AUTH_SECRET_KEY=${REACT_APP_ANGEL_AUTH_SECRET_KEY}
-        - REACT_APP_KADO_API_KEY=${REACT_APP_KADO_API_KEY}
         - REACT_APP_NETWORK=${REACT_APP_NETWORK}
         - REACT_APP_JUNO_LCD_NODE=${REACT_APP_JUNO_LCD_NODE}
```

Comment:
```suggestion
```
Should we remove this line too? 

There are other uses of the same env. var in [.github/workflows/javascript.yml](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/auth-token/.github/workflows/javascript.yml)

Reply:
fixed